src/main/java/com/sap/cloudfoundry/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -237,6 +237,7 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(CloudControllerRestClientImpl.class);
     private static final long JOB_POLLING_PERIOD = TimeUnit.SECONDS.toMillis(5);
     private static final int MAX_CHAR_LENGTH_FOR_PARAMS_IN_REQUEST = 4000;
+    private static final List<String> CHARS_TO_ENCODE = List.of(",");
```

Comment:
I suppose that list is used because in future could be necessary to encode more characters, so do you think that this constants could be placed somewhere else?

Reply:
Yes, you are right about the list. However, I am not sure if there is more suitable place for this constant as it is related to the query params in rest operations. I do not think it is good to be in util class.